test/iterators.jl

```diff
@@ -540,10 +540,73 @@ end
 	@test isempty(collect(zip(b,a)))
 	@test !isempty(a)
     end
+    let a = Iterators.Stateful("a"), b = "", c = Iterators.Stateful("c")
+       @test isempty(collect(zip(a,b,c)))
```

Comment:
Incorrect indentation for these

Reply:
Well spotted!

Reply:
What can I say, I have an eye for pedantry :P